src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -44,15 +45,54 @@ For example, if your app is already installed:
 
 <div class="w-clearfix"></div>
 
-### Supported app types you can check
+To use the `getInstalledRelatedApps()` API, you need to tell your app about
+your site, then tell your site about your app. Once you've defined the
+relationship between the two, you can check if the app is installed.
 
-Your website can check if your:
+### Supported app types you can check
 
-* [Android app](#check-android) is installed
-* [Windows (UWP) app](#check-windows) is installed
-* Progressive Web App is installed, if it's running in the
-  [same scope](#check-pwa-in-scope) or in a
-  [different scope](#check-pwa-out-of-scope)
+<div class="w-table-wrapper">
+  <table class="w-table--top-align">
+    <thead>
+      <tr>
+        <th>Can check for your</th>
+        <th>Can check from</th>
```

Comment:
I'm not crazy about these heading. It looks like you're using a sentence style headings. Though that's a perfectly fine thing to do, these headings feel clunky to me. What about more functional headings like 'App type' and 'Checkable from'?

Reply:
Good suggestion, changed to app type/checkable from.